packages/grid/test/selection.test.js

```diff
@@ -509,4 +510,52 @@ describe('multi selection column', () => {
 
     expect(grid.selectedItems).to.eql(grid.items);
   });
+
+  it('should select items on mouse drag when selectRowsByDragging is enabled', () => {
+    grid.items = ['foo', 'bar', 'baz', 'qux', 'quux', 'corge'];
+    flushGrid(grid);
+    rows = getRows(grid.$.items);
+
+    selectionColumn.selectRowsByDragging = true;
+
+    const sourceCell = getCellContent(getRowCells(rows[0])[0]);
+    const sourceCellRect = sourceCell.getBoundingClientRect();
+
+    const targetCell = getCellContent(getRowCells(rows[3])[0]);
+    const targetCellRect = targetCell.getBoundingClientRect();
+
+    const clock = sinon.useFakeTimers();
```

Comment:
Sinon stubs should always be set up in `beforeEach`, and then restored in `afterEach`. Otherwise, if anything causes the test to not execute the restore, all following tests will be affected as well.

Reply:
Done